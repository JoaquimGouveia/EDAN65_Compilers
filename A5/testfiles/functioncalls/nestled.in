int main(){
    print(f(g()));
    return 0;
}

int f(int i){
    return i;
}

int g(){
    return 3;
}
