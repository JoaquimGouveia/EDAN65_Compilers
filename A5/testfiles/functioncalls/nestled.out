Function: main
f
print
g
---------------
Function: f
---------------
Function: g
---------------
