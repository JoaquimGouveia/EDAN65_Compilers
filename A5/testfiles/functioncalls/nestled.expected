Function: main
print
f
g
---------------
Function: f
---------------
Function: g
---------------
